microprofile/src/main/java/io/quarkus/ts/openshift/microprofile/HelloClient.java

```diff
@@ -13,7 +13,8 @@
 
 import static java.util.concurrent.CompletableFuture.completedFuture;
 
-@RegisterRestClient(baseUri = "http://microprofile-test:8080/")
+// TODO https://github.com/quarkusio/quarkus/issues/16844
+@RegisterRestClient(baseUri = "http://localhost:8080/")
```

Comment:
Why is this change needed?

Reply:
There is an issue related to `quarkus.application.name` and rest-client. 
https://github.com/quarkusio/quarkus/issues/16844